### DISCUSSION TITLE
Zoom migration

### DISCUSSION THREAD

**Discussion by hajrullinbulat** [twilio/twilio-video-app-react#840] [Q&A]
I've centered my project around this repository, and I'm curious to know if anyone has undertaken the task of transitioning the project from Twilio to Zoom. If so, I'd appreciate hearing about any challenges encountered during the process. Additionally, if you have any recommendations or insights to expedite the migration, I'd be eager to learn from your experience.

**Comment by llamorinlabs** [comment]
@tommygaessler

**Comment by tommygaessler** [comment]
Hey @llamorinlabs , thank you for looping me in.

@hajrullinbulat , checkout the Video SDK UI toolkit: https://github.com/zoom/videosdk-ui-toolkit-web

Here is a React sample: https://github.com/zoom/videosdk-ui-toolkit-react-sample
